web-jehlomat/src/Components/Form/PridatUzivateleForm.tsx

```diff
@@ -0,0 +1,157 @@
+import { FC } from 'react';
+import { Formik } from 'formik';
+import { useMediaQuery } from '@mui/material';
+import TextInput from '../Inputs/TextInput/TextInput';
+import { FormItemLabel, FormItemDescription } from '../../utils/typography';
+import { FormWrapper, FormItem } from '../Form/Form';
+import PrimaryButton from '../Buttons/PrimaryButton/PrimaryButton';
+import Box from '@mui/material/Box';
+import { useHistory } from 'react-router-dom';
+import { authorizedAPI } from '../../config/baseURL';
+import { AxiosResponse } from 'axios';
+import * as yup from 'yup';
+import styled from '@emotion/styled';
+import { primaryDark } from '../../utils/colors';
+import { media } from '../../utils/media';
+import { LINKS } from 'utils/links';
+
+interface Props {}
+
+interface Values {
+    email: string;
+}
+
+const validationSchema = yup.object({
+    email: yup.string().email('Vlož validní email').required('Email je povinné pole'),
+});
+
+const ModalFormWrapper = styled(FormWrapper)<{ mobile?: boolean }>`
+    ${props =>
+        props.mobile &&
+        `
+        height: calc(100vh - 100px);
+        justify-content: flex-start;
+    `}
+`;
+
+const ModalFormItem = styled(FormItem)<{ mobile?: boolean }>`
+    font-family: Roboto;
+    margin: 15px 10px;
+    text-align: center;
+    width: 90%;
+
+    @media (min-width: 700px) {
+        width: 85%;
+    }
+    ${props =>
+        props.mobile &&
+        `
+        width: 90%;
+    `}
+
+`;
+
+const ModalFormItemLabel = styled(FormItemLabel)<{ mobile?: boolean }>`
+    text-transform: none;
+    ${props =>
+        props.mobile &&
+        `
+        font-weight: 700;
+        font-size: 18px;
+        text-align: left;
+    `}
+`;
+
+const ModalFormItemDescription = styled(FormItemDescription)<{ mobile?: boolean, below?: boolean }>`
+    color: ${primaryDark};
+    text-align: center;
+    font-size: 18px;
+    @media (min-width: 700px) {
+        width: 85%;
+    }
+    ${props =>
+        props.mobile &&
+        `
+        margin: 3rem 0;
+        font-weight: 500;
+        line-height: 24px;
+        text-align: left;
+        width: 90%;
+
+    `}
+    ${props =>
+        (props.below && props.mobile) &&
+        `
+        font-size: 16px;
+        margin: 0 0 2rem;
+    `}
+`;
+
+
+const PridatUzivateleModal: FC<Props> = () => {
+    const isMobile = useMediaQuery(media.lte('mobile'));
+
+    let history = useHistory();
```

Comment:
```suggestion
    const history = useHistory();
```
To be consistent also, the variable is not later modified.